cms/schemas/podcast.js

```diff
@@ -19,6 +19,11 @@ export default {
       },
       validation: (Rule) => Rule.required(),
     },
+    {
+      name: "rssId",
```

Comment:
```suggestion
      name: "rssId",
      description:
        "6-digit ID used to construct the URL to this podcast episode on rss.com",
```

Reply:
Thanks @Atmosfearful, yeah for sure the description will be helpful - updated.